tests/testthat/examples/Test_2023_1/output/taxonomic_updates.csv

```diff
@@ -1,48 +1,48 @@
-Test_2023_1,Ixora biflora,Ixora biflora,,Ixora biflora,accepted,https://id.biodiversity.org.au/node/apni/2915627
-Test_2023_1,Lasianthus strigosus,Lasianthus strigosus,,Lasianthus chlorocarpus,misapplied,https://id.biodiversity.org.au/instance/apni/7839722
-Test_2023_1,Litsea leefeana,Litsea leefeana,,Litsea leefeana,accepted,https://id.biodiversity.org.au/node/apni/2921063
-Test_2023_1,Macaranga tanarius,Macaranga tanarius,,Macaranga tanarius,accepted,https://id.biodiversity.org.au/node/apni/2902044
-Test_2023_1,Mallotus mollissimus,Mallotus mollissimus,,Mallotus mollissimus,accepted,https://id.biodiversity.org.au/node/apni/2904445
-Test_2023_1,Medicosma sessiliflora,Medicosma sessiliflora,,Medicosma sessiliflora,accepted,https://id.biodiversity.org.au/node/apni/2914896
-Test_2023_1,Melastoma cyanoides,Melastoma cyanoides,,Melastoma cyanoides,accepted,https://id.biodiversity.org.au/node/apni/2895184
-Test_2023_1,Melicope elleryana,Melicope elleryana,,Melicope elleryana,accepted,https://id.biodiversity.org.au/node/apni/2913641
-Test_2023_1,Myristica insipida,Myristica insipida,,Myristica insipida,accepted,https://id.biodiversity.org.au/node/apni/2902429
-Test_2023_1,Neolitsea dealbata,Neolitsea dealbata,,Neolitsea dealbata,accepted,https://id.biodiversity.org.au/node/apni/2910025
-Test_2023_1,Pittosporum rubiginosum,Pittosporum rubiginosum,,Pittosporum rubiginosum,accepted,https://id.biodiversity.org.au/node/apni/2887772
-Test_2023_1,Polyscias australiana,Polyscias australiana,,Polyscias australiana,taxonomic synonym,https://id.biodiversity.org.au/instance/apni/941828
-Test_2023_1,Psychotria dallachiana,Psychotria dallachiana,,Psychotria dallachiana,accepted,https://id.biodiversity.org.au/node/apni/2890013
-Test_2023_1,Psychotria sp Utchee Creek,Psychotria sp. Utchee Creek (H.Flecker NQNC5313),species,Psychotria sp. Utchee Creek (H.Flecker NQNC5313),accepted,https://id.biodiversity.org.au/taxon/apni/51446727
-Test_2023_1,Quassia baileyana,Quassia baileyana,,Samadera baileyana,nomenclatural synonym,https://id.biodiversity.org.au/instance/apni/897542
-Test_2023_1,Rhodomyrtus trineura,Rhodomyrtus trineura,,Rhodomyrtus trineura,accepted,https://id.biodiversity.org.au/taxon/apni/51441176
-Test_2023_1,Rockinghamia angustifolia,Rockinghamia angustifolia,,Rockinghamia angustifolia,accepted,https://id.biodiversity.org.au/node/apni/2921179
-Test_2023_1,Syzygium,Syzygium,,,,
-Test_2023_1,Syzygium sayeri,Syzygium sayeri,,Syzygium sayeri,accepted,https://id.biodiversity.org.au/node/apni/2888968
-Test_2023_1,Testupdated sp4,Testupdated sp4 replaced,species,,,
-Test_2023_1,Trema aspera,Trema aspera,,Trema tomentosa var. aspera,nomenclatural synonym,https://id.biodiversity.org.au/instance/apni/856997
-Test_2023_1,testupdated sp3,testupdated sp3 replaced,species,,,
+dataset_id,original_name,aligned_name,taxonomic_resolution,taxon_name,aligned_name_taxon_id,aligned_name_taxonomic_status
```

Comment:
For all of these, is the only change that `aligned_name_taxon_id` and `aligned_name_taxonomic_status` columns swapped order? And if so, do we want that? I'm actually inclined to change the code that selects what columns to maintain and keep the original order in these files. Because I've been tending to keep the identifiers at the far right of tables

Reply:
Yes I think that's the only change. Will have to re-update `build_update_taxon_list()` in austraits.build as well.

Reply:
Nevermind it would have to be in [line 1917](https://github.com/traitecoevo/traits.build/blob/93cfd3ef1cb57df7e93bc4d844128a0f6ffb5e49/R/process.R#L1917-L1918) in process.R - would need to list the `align` columns by name instead of `contains`. But we don't want to change the output structure again so will leave it as is.